src/Main.cc

```diff
@@ -234,7 +233,7 @@ tclReadlineInit(Tcl_Interp *interp)
     "eval $auto_index(::tclreadline::ScriptCompleter)",
     "::tclreadline::readline builtincompleter true",
     "::tclreadline::readline customcompleter ::tclreadline::ScriptCompleter",
-    "proc ::tclreadline::prompt1 {} { return \"openroad " OPENROAD_VERSION "> \" }",
+    "proc ::tclreadline::prompt1 {} { return \"openroad > \" }",
```

Comment:
might want to remove the space between `openroad >` to `openroad>`

Reply:
changed on 114147c